src/metatrain/utils/data/readers/readers.py

```diff
@@ -135,28 +75,51 @@ def read_systems(
         determined from the file extension.
     :returns: list of systems stored in double precision
     """
-    return _base_reader(target="systems", filename=filename, reader=reader)
+    return _call_base_systems_reader(filename=filename, reader=reader)
 
 
-def read_virial(
-    filename: str,
-    target_value: str = "virial",
-    reader: Optional[str] = None,
-) -> List[TensorBlock]:
-    """Read virial informations from a file.
+def _call_base_target_reader(
```

Comment:
I this function only used in `read_systems`. If yes I think we don't need an extra function definition.